src/test/groovy/com/meltwater/docker/compose/DockerComposeTest.groovy

```diff
@@ -24,7 +24,7 @@ class DockerComposeTest extends Specification {
 
     def 'test simple compose.up'() {
         when:
-            List<InspectData> inspectData = compose.up()
+            List<InspectData> inspectData = compose.up(Recreate.DEFAULT)
```

Comment:
The argument is optional right?

Reply:
Yes, but for some reason I couldn't get Groovy play nicely with the optional parameters of Kotlin.